### PR TITLE
Docs: update automated testing learning objectives

### PR DESCRIPTION
**Pull request by signekb**
## Description

Updates learning objectives of the automated testing session. What do you think? :)

### CODE REVIEW

File: sessions/automated-testing.qmd [hunk 1]
```diff
@@ -12,7 +12,13 @@
 
 **Session objectives**:
 
--   Deploy a GitHub Action that runs a continuous integration workflow
+The overall objective for this session is to:
+-   Explain what *automated testing* means when developing packages, 
+    and deploy simple GitHub Actions to automate tests
```
Comment (lwjohnst86) [inline sessions/automated-testing.qmd:17]: "deploy" is a fairly "jargon" term and doesn't precisely say what they will do. Something like "create and upload".

Comment (signekb) [reply sessions/automated-testing.qmd:17]: @lwjohnst86 Hm, I'm not sure that "upload" covers what I want to say. How about:
```suggestion
    and create simple GitHub Actions that will automatically run tests when new code is pushed to GitHub
```

Comment (lwjohnst86) [reply sessions/automated-testing.qmd:17]: Fantastisk! 🎉

File: sessions/automated-testing.qmd [hunk 2]
```diff
@@ -12,7 +12,13 @@
...
 **Session objectives**:
 
--   Deploy a GitHub Action that runs a continuous integration workflow
+The overall objective for this session is to:
+-   Explain what *automated testing* means when developing packages, 
+    and create simple GitHub Actions that will automatically run tests when new code is pushed to GitHub
+
+More specific objectives are to:
+-   Deploy a GitHub Action that runs CMD Check
+-   Deploy a GitHub Action that runs test coverage
```
Comment (signekb) [inline sessions/automated-testing.qmd:21]:
```suggestion
-   Create a GitHub Action that runs CMD Check
-   Create a GitHub Action that runs test coverage
```

**Pull request merged**